test/ppx_expect/main.ml

```diff
@@ -0,0 +1,78 @@
+let test_directory = "cases"
+let test_cases = [
```

Comment:
It's not necessary, but I think it would be nice to get the test list by listing the directory.